src/common/components/cards/rich-resolution-content.tsx

```diff
@@ -89,6 +89,174 @@ export const RichResolutionContent = NamedFC<RichResolutionContentProps>(
                     </span>
                 );
             }
+            case 'android/atfa/ClassNameCheck': {
+                return (
+                    <span>
+                        Provide a <Markup.Code>className</Markup.Code> value within the element's{' '}
+                        <LinkComponent href="https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html">
+                            AccessibilityNodeInfo
+                        </LinkComponent>{' '}
+                        that:
+                        <ol>
+                            <li>Closely describes the element's function,</li>
+                            <li>Matches a class that extends android.view.View,</li>
```

Comment:
Referencing the word doc, 
```suggestion
                            <li>Matches a class that extends <Markup.Code>android.view.View</Markup.Code>,</li>
```

Reply:
Nice catch! Fixed.